Gemfile

```diff
@@ -49,9 +49,9 @@ gem 'cancancan'
 gem 'draper'
 gem 'pundit'
 
-group :development, :test do
+group :development, :production, :test do
```

Comment:
🙅‍♂ The production should not need rspec. We can discuss it

Reply:
I know, but rails thinks that heroku is a production environment so I needed to add that in order to push it on staging